server/src/main/java/org/apache/iotdb/db/qp/executor/PlanExecutor.java

```diff
@@ -1261,46 +1259,26 @@ private boolean createMultiTimeSeries(CreateMultiTimeSeriesPlan multiPlan)
         multiPlan.getResults().put(i, RpcUtils.getStatus(e.getErrorCode(), e.getMessage()));
       }
     }
-    if (!multiPlan.getResults().isEmpty()) {
-      throw new BatchProcessException(multiPlan.getResults().values().toArray(new TSStatus[0]));
-    }
-    return true;
+    throw new BatchProcessException(multiPlan.getResults().values().toArray(new TSStatus[0]));
   }
```

Comment:
Why throw BatchProcessException always?
if all CreateTimeSeriesPlans execute success, we should return true.
besides, the createMultiTimeSeries's bug have been fix in #2698